packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "vscode": "1.1.4"
   },
+  "extensionDependencies": ["salesforce.salesforcedx-vscode-apex"],
```

Comment:
VS Code won't let the user uninstall an extension that another extension depends on. If you try to uninstall vscode-apex-debugger, it will ask if you only want to uninstall that extension or also its dependencies.